api/src/main/java/org/openmrs/module/fhir2/providers/r3/PersonFhirResourceProvider.java

```diff
@@ -62,6 +72,34 @@ public Person getPersonById(@IdParam @NotNull IdType id) {
 		return Person30_40.convertPerson(person);
 	}
 	
+	@Create
+	public MethodOutcome createPerson(@ResourceParam Person person) {
+		return FhirProviderUtils.buildCreate(personService.create(Person30_40.convertPerson(person)));
+	} 
+
+	@Update
+	@SuppressWarnings("unused")
+	public MethodOutcome updatePerson(@IdParam IdType id, @ResourceParam Person person) {
+		if (id == null || id.getIdPart() == null) {
+			throw new InvalidRequestException("id must be specified to update");
+		}
+
+		person.setId(id.getIdPart());
+
+		return FhirProviderUtils.buildUpdate(personService.update(id.getIdPart(), Person30_40.convertPerson(person)));
+	}
+
+	@Delete
+	@SuppressWarnings("unused")
+	public OperationOutcome deletePerson(@IdParam @NotNull IdType id) {
+		org.hl7.fhir.r4.model.Person person = personService.delete(id.getIdPart());
+		if (person == null) {
+			throw new ResourceNotFoundException("Could not find person to delete with id " + id.getIdPart());
+		}
+		return OperationOutcome30_40.convertOperationOutcome(FhirProviderUtils.buildDelete(person));
```

Comment:
```java
return FhirProviderUtils.buildDelete(Person30_40.convertPerson(person)));
```

Reply:
@corneliouzbett i have fixed it ,,,kindly review